rasa/test.py

```diff
@@ -124,6 +130,84 @@ def test_nlu(model: Optional[Text], nlu_data: Optional[Text], kwargs: Optional[D
         )
 
 
+def test_compare_nlu(
```

Comment:
Nice to have: The method is quite long, can you split it up into multiple methods?